src/app/newProject/page.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
```

Comment:
This route is probably going to be `projects/:id`.
It would not be a new project but rather an existing one.
Additionally we don't want capitals or camelcase in our URLs. So, might you create a placeholder ticket saying to refactor the `newProject` page to be output at the route `projects/:id` once the API client is ready in the front-end? 

Thanks!